submissions/dummy_submission/the-nuclear-options.md

```diff
@@ -0,0 +1,150 @@
+
+* **Per reactor, build only as many heat exchangers and steam turbines as you need** - These machines account for the majority of the UPS drain (around 80% of an optimized setup), likely because of just how many you need but also because of the calculations they need to do. *u/warbaque*'s various tests note that the utilization percentage of steam turbines affects their UPS drain, but fewer turbines at 100% utilization perform better than more turbines at lower utilization. Meanwhile, some mods offer higher tier versions of these buildings so that you need fewer of them.
+
+* **Minimize heat pipe usage** - The game's heat manager was found to be the other big UPS eater (around 20% of an optimized setup). The number of heat connections is main factor to consider, meaning that heat pipes should be as short as possible and be only one tile thick.
+
+* **Refuel reactor arrays using bots or clocked belts** - While bots may sounds less UPS-efficient than belts, the issue with using belts is inserter tracking, which is when the inserters try to catch moving items on belts. This issue occurs a lot when you have a long belt servicing many reactors. It causes an extra 6% of UPS overhead to an optimized setup. The way around the issue is to use bots to deliver fuel cells. Alternatively, you can use a transport belt that is not looping and inserters that are "clocked", meaning that they are timed using circuit networks such that they all take from the belt at once and then wait for the belt to fill up before they are allowed to take again.
+
+* **Minimize fluid entity usage, particularly pumps** - In an optimized setup, the fluid system accounts for less than 1% of UPS drain. On the other hand, a setup with pressurized steam pipes (one pump per two exchangers) had an extra UPS overhead of 15%. Hence it is preferable to use very few pumps, while afterwards one may consider removing storage tanks (starting from steam storage).
+
+* **Prioritize optimizing what takes up the most time** - In Factorio, the heat manager, the fluid manager, and the electric network are simulated in parallel. Therefore, the slowest of these three factors causes the UPS bottleneck for the group while the other two have little to no impact. You can check this by opening the debug settings with the `F4` key and selecting "show-time-usage" and looking at which factor lists the largest numbers for update times. Based on this you can determine what to add or remove from your designs. For example, if the fluid manager costs comparatively little time, you can add more steam storage without worry.
```

Comment:
```suggestion
* **Prioritize optimizing what takes up the most time** - In Factorio, the heat manager, the fluid manager, and the electric network are simulated in parallel. Therefore, the slowest of these three factors causes the UPS bottleneck for the group while the other two have little to no impact. You can check this by opening the debug settings with the `F4` key, selecting "show-time-usage" and looking at which factor lists the largest numbers for update times. Based on this you can determine what to add or remove from your designs. For example, if the fluid manager costs comparatively little time, you can add more steam storage without worry.
```

Reply:
Maybe `and looking at which factor lists the largest numbers for update times.` should be rephrased, but it may just be me.